src/display/api.js

```diff
@@ -952,15 +952,51 @@ var PDFPageProxy = (function PDFPageProxyClosure() {
 
     /**
      * @param {getTextContentParameters} params - getTextContent parameters.
+     * @return {ReadableStream} ReadableStream to read textContent chunks.
+     */
+    streamTextContent(params) {
```

Comment:
`streamTextContent(params = {}) {`, otherwise `params` won't be optional (as it is for backwards compatibility reasons in `getTextContent` below).